frontend/src/multiAdd/MultiAddItem.tsx

```diff
@@ -38,20 +37,18 @@ function MultiAddItem(props: IMultiAddItemProps) {
 
   return (
     <div className={styles.itemContainer}>
-      <TextField
-        defaultValue={props.name}
-        size="small"
-        disabled={!isAdmin}
-        InputProps={{
-          readOnly: true,
-        }}
-      />
+      {!isAdmin &&
+        <Chip
+          label={props.name}
+        />
+      }
       {
         isAdmin &&
-        <DeleteIcon
-        className={styles.deleteIcon}
-        onClick={() => props.onDelete(props.name)}
-      />
+        <Chip
+          label={props.name}
+          disabled={!isAdmin}
```

Comment:
this disabled condition should be implicitly false but it's more readable this way

also, the syntax of `{ isAdmin && <a> } ... { !isAdmin && <b> }` could be combined to `{ isAdmin ? <a> : <b> }`

but not critical 😃